app/notices/shop_employee_notices/employee_waiver_confirm_notice.rb

```diff
@@ -0,0 +1,19 @@
+class ShopEmployeeNotices::EmployeeWaiverConfirmNotice < ShopEmployeeNotice
+  attr_accessor :census_employee
+
+  def deliver
+    build
+    generate_pdf_notice
+    employee_appeal_rights_attachment
+    attach_envelope
+    non_discrimination_attachment
+    upload_and_send_secure_message
+    send_generic_notice_alert
+  end
+
+  def append_data
+    notice = PdfTemplates::PlanYear.new({
```

Comment:
Use hbx_enrollment template as it seems to be more related to the waiver.
And also make sure to add the `waived_on` field on hbx_enrollment template.